### PR TITLE
Update edit-viva-home.md to retrieve the current state of home site experience before updating

### PR DESCRIPTION
**Pull request by cts-mbiswas**
IT admins are asking for a way to retrieve the current state of the default landing to the home site before making a change. IT admins always do not have Viva Connections enabled as some orgs have IT managed by partners . Proposing a change to retrieve the current state of default landing to the home site before making a change.

### CODE REVIEW

File: Viva/connections/edit-viva-home.md [hunk 1]
```diff
@@ -220,7 +220,12 @@ If your organization already has a SharePoint home site and you want to keep it
 
 2.	Connect to SharePoint as a [Global Administrator](/microsoft-365/admin/add-users/about-admin-roles) in Microsoft 365. To learn how, see [Getting started with SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online).
 
-3.	Run `Set-SPOHomeSite -HomeSiteUrl <homesiteURL> -VivaConnectionsDefaultStart <$true/$false>`
+3.	Run `Get-SPOHomesite|fl and look for the value of VivaConnectionsDefaultStart <$true/$false>` for the current landing expereince.
```
Comment (JohanFreelancer9) [inline Viva/connections/edit-viva-home.md:223]:
```suggestion
3.	Run `Get-SPOHomesite|fl and look for the value of VivaConnectionsDefaultStart <$true/$false>` for the current landing experience.
```

File: Viva/connections/edit-viva-home.md [hunk 2]
```diff
@@ -229,8 +234,11 @@ If your organization already has a SharePoint home site and you want to keep it
 <br>
 Contoso’s home site URL is https://contoso.sharepoint.com/sites/homeSite in this example.
 
-**To set the default landing to the home site:**
+**To retrieve the default landing to the home site:**
+$Site = Get-SPOHomeSite
```
Comment (JohanFreelancer9) [inline Viva/connections/edit-viva-home.md:238]:
```suggestion
`$Site = Get-SPOHomeSite`
```

File: Viva/connections/edit-viva-home.md [hunk 3]
```diff
@@ -229,8 +234,11 @@ If your organization already has a SharePoint home site and you want to keep it
...
+$Site.VivaConnectionsDefaultStart
```
Comment (JohanFreelancer9) [inline Viva/connections/edit-viva-home.md:239]:
```suggestion
`$Site.VivaConnectionsDefaultStart`
```

File: Viva/connections/edit-viva-home.md [hunk 4]
```diff
@@ -229,8 +234,11 @@ If your organization already has a SharePoint home site and you want to keep it
...
 
+**To set the default landing to the home site:**
```
Comment (JohanFreelancer9) [inline Viva/connections/edit-viva-home.md:241]:
```suggestion
**To set the default landing to the home site:**

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cts-mbiswas, kindly commit the suggested changes. Many thanks.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by cts-mbiswas** [comment]
Hello Team

I was wanting to follow up on this as I was not able to see the suggested changes, but now I see the feedback item closed with seems an action step pending one me. Can you please guide me on the next steps please  so that we can get the recommended changes published.

Regards,
Manas

**Comment by JohanFreelancer9** [comment]
@scanum

**Comment by cts-mbiswas** [comment]
Thanks JohanFreelancer9 - including my original message as well.


From: Manas Kumar Biswas
Sent: Wednesday, October 4, 2023 3:03 AM
To: 'MicrosoftDocs/Viva' ***@***.******@***.***>; MicrosoftDocs/Viva ***@***.******@***.***>
Cc: Assign ***@***.******@***.***>; Manas Kumar Biswas ***@***.******@***.***>
Subject: RE: [MicrosoftDocs/Viva] Update edit-viva-home.md to retrieve the current state of home site experience before updating (PR #505)

Hello Team

I was wanting to follow up on this as I was not able to see the suggested changes, but now I see the feedback item closed with seems an action step pending one me. Can you please guide me on the next steps please  so that we can get the recommended changes published.

Regards,
Manas

**Comment by cts-mbiswas** [comment]
Hello Team,

Any pointers regarding my request below..

Regards,
Manas

From: Manas Kumar Biswas ***@***.***>
Sent: Wednesday, October 4, 2023 3:03 AM
To: MicrosoftDocs/Viva ***@***.***>; MicrosoftDocs/Viva ***@***.***>
Cc: Assign ***@***.***>; Manas Kumar Biswas ***@***.***>
Subject: RE: [MicrosoftDocs/Viva] Update edit-viva-home.md to retrieve the current state of home site experience before updating (PR #505)

Hello Team

I was wanting to follow up on this as I was not able to see the suggested changes, but now I see the feedback item closed with seems an action step pending one me. Can you please guide me on the next steps please  so that we can get the recommended changes published.

Regards,
Manas

**Pull request closed without merging**